src/sidebar/components/navigation/navigation.tpl.js

```diff
@@ -0,0 +1,40 @@
+const navigation = () => {
+
+	let style = document.createElement('link');
+	style.setAttribute('rel', "stylesheet");
+	style.setAttribute('href', "components/navigation/navigation.css");
```

Comment:
can you use the css import instead (e.g. see `ensign`) - it's less code and defines the dependency more explicitly